rules/windows/registry/registry_set/registry_set_policies_associations_tamper.yml

```diff
@@ -0,0 +1,42 @@
+title: Modify Attachment Manager Settings - Associations
+id: a9b6c011-ab69-4ddb-bc0a-c4f21c80ec47
+status: experimental
+description: Detects tampering with attachment manager settings policies associations to lower the default file type risks (See reference for more information)
+author: Nasreddine Bencherchali
+references:
+    - https://support.microsoft.com/en-us/topic/information-about-the-attachment-manager-in-microsoft-windows-c48a4dcd-8de5-2af5-ee9b-cd795ae42738
+    - https://www.virustotal.com/gui/file/2bcd5702a7565952c44075ac6fb946c7780526640d1264f692c7664c02c68465
+date: 2022/08/01
+logsource:
+    category: registry_set
+    product: windows
+detection:
+    root:
+        EventType: SetValue
+        TargetObject|startswith:
```

Comment:
I'd remove the Root of the key (e.g. `HKLM` or `HKEY_LOCAL_MACHINE`) and use `contains`